github3/repos/repo.py

```diff
@@ -620,6 +620,30 @@ def create_blob(self, content, encoding):
         return sha
 
     @requires_auth
+    def create_branch(self, name, sha=None):
+        """Create a branch. Returns False if branch doesn't exist
+        or not authorized.
+
+        :param str branch: (required), the branch to create.
+        :param str|commit sha: (optional), the commit to base
+                the branch from, otherwise master head.
+
+        :returns: bool -- True if successful, False otherwise
+        """
+        if isinstance(sha, Commit):
+            sha = commit.sha
+        elif not isinstance(sha, basestring):
```

Comment:
Drat! Whats the cross environment way to check if something is string or unicode in that case?